assets/sass/partials/objects/_type-scale.scss

```diff
@@ -1,59 +1,54 @@
-$base: 16px;
+$base: 18px;
 $base-lg: 18px;
```

Comment:
Probably don't need the `$base-lg` anymore.

Reply:
This has now been removed